README.md

```diff
@@ -117,7 +158,16 @@ Using the object literal notation, Define an object called student and give it t
 
 Your Answer:
 ```js
-// write code here
+
+var student = function () {
+	return {
+       name: "Mara Sanders",
+       age: "29+",
+       method : function sayHello () {
+         console.log ("Hi, my name is"+student.name);
+       }  
+    };
+
 ```
```

Comment:
Good example of a constructor function, but the question was just asking for an object literal, i.e. :
 
```js
var student = {
  name: "Mara",
  age: "29",
  method: function () {
    console.log("Hi, my name is " + this.name)
  }
}
```